### ISSUE TITLE
[EN] Possible Grammar Error or the Sentense has Error

### ISSUE THREAD

**Issue by JonathanSum** [Atcold/pytorch-Deep-Learning#398]
In model distillation we take the pre-trained network and **use the labels the network** predicted in a softer fashion to generate labels for our images.

It is not just the comma. I don't really understand: use the labels the network

**Comment by Atcold** [comment]
[…] use the labels [that] the network […]

The net predicts soft labels which are used to train the smaller model.
The *that* is not necessary in English.
